wagtail/wagtailsearch/backends/elasticsearch.py

```diff
@@ -398,6 +399,38 @@ def __repr__(self):
 
 
 class ElasticsearchSearchResults(BaseSearchResults):
+    def __init__(self, *args, **kwargs):
+        super(ElasticsearchSearchResults, self).__init__(*args, **kwargs)
+        self.highlight_params = {}
+
+    def highlight(self, **kwargs):
```

Comment:
Also, I noticed that the parameters passed to highlight are all just sent to Elasticsearch. To help us support highlighting for more backends in the future, maybe we should do something similar to the ``SearchFields`` API and support the common parameters in the API but allow customisation by passing a ``dict`` in a kwarg called ``es_extra``.